### DISCUSSION TITLE
Mediabox starts from arbitary point

### DISCUSSION THREAD

**Discussion by yuyang1995** [pymupdf/PyMuPDF#1403] [Looking for help]
I want to move the content in a page to the right a little bit. So I need to increase the white margin on the left and reduce the white margin on the right. A possible way is to set the media box from some minus value, like (-10, 0, 489, 709). But it says that 'mediabox must start at (0,0)'. Is there any solution?

**Comment by JorjMcKie** [comment]
I know I had a good reason when I introduced that restriction - just can't recall what that was.
But you can always use the low-level interface to enforce this:

``doc.xref_set_key(page.xref, "MediaBox", "[-10 0 489 709]")``
